### PR TITLE
fix(Paprika): UXD-1787 fix doc parser

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️

Example for a js package: https://github.com/acl-services/paprika/tree/UXD-1787-fix-parser/packages/Button
Example for a ts package:  https://github.com/acl-services/paprika/tree/UXD-1787-fix-parser/packages/Link

### Updates 📦
If you have changed a component's source code (not stories, specs, or docs), before merging your branch run `yarn changeset`. This will prompt you to:
- indicate if changes were patch/minor/major for each modified package
- enter a release message

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/Takeover/src/components/Content/Content.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
   border-radius: ${tokens.card.borderRadius};
   box-shadow: ${tokens.card.shadow};
   margin: ${spacer(2)};
   padding: ${spacer(2)};
 `;
 
-Content.displayName = "Takeover.Content";
+export default function Content(props) {
+  return <StyledContent {...props} />;
+}
```
Comment (allison-c) [inline packages/Takeover/src/components/Content/Content.js:16]: For passing react-docgen

File: packages/Link/src/Link.tsx [hunk 2]
```diff
@@ -67,5 +82,16 @@ function Link(props: LinkProps): JSX.Element {
...
+  a11yText: null,
+  children: null,
+  isExternalLink: false,
+  hasNoUnderline: false,
+  hasTruncation: false,
+  isSubtle: false,
+  isDark: false,
+  isMenu: false,
+  maxWidth: "100%",
+} as Partial<LinkProps>;
```
Comment (allison-c) [inline packages/Link/src/Link.tsx:95]: Example for passing both tsc and react-docgen

File: scripts/reactDocHelpers/readmeTemplate.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+${content}
+
+## Links
+- [Storybook Showcase](https://paprika.highbond.com/?path=/story/${getStoryUrlPrefix(displayName)}--showcase)
+- [GitHub source code](https://github.com/acl-services/paprika/tree/master/packages/${displayName}/src)
+- [Create GitHub issue](https://github.com/acl-services/paprika/issues/new?label=[]&title=${name}%20[help]:%20your%20short%20description&body=${githubIssueBody})
+- [CHANGELOG](https://github.com/acl-services/paprika/tree/master/packages/${displayName}/CHANGELOG.md)
+`;
+
+module.exports = readmeTemplate;
```
Comment (allison-c) [inline scripts/reactDocHelpers/readmeTemplate.js:53]: Moved from `readmeFile.js`

**Review by AndreyChernykh — APPROVED** [review]
LGTM

**Pull request merged**